### PR TITLE
Moving to IBAnimatable organization

### PR DESCRIPTION
**Pull request by JakeLin**
@tbaranes please have a look when you have time. Do you think something is missing, or we can merge it and change the settings to move?

### CODE REVIEW

File: Documentation/APIs.md [hunk 1]
```diff
@@ -32,8 +32,8 @@ To use `IBAnimatable`, we can drag and drop a UIKit element and connect it with
 #### `BlurDesignable`
 | Property name | Data type | Description |
 | ------------- |:-------------:| ----- |
-| blurEffectStyle | Optional&lt;String> | Support three different blur effects: `ExtraLight`, `Light` and `Dark`, also can be found in emum [`BlurEffectStyle `](https://github.com/JakeLin/IBAnimatable/blob/master/IBAnimatable/BlurEffectStyle.swift). The look of blur effect in Interface Builder is different from Simulator or device. |
-| vibrancyEffectStyle | Optional&lt;String> | Support three different blur effects: `ExtraLight`, `Light` and `Dark`, also can be found in emum [`BlurEffectStyle `](https://github.com/JakeLin/IBAnimatable/blob/master/IBAnimatable/BlurEffectStyle.swift). Once specify the Vibrancy effect style, all subviews will apply this vibrancy effect. |
+| blurEffectStyle | Optional&lt;String> | Support three different blur effects: `ExtraLight`, `Light` and `Dark`, also can be found in emum [`BlurEffectStyle `](IBAnimatable/BlurEffectStyle.swift). The look of blur effect in Interface Builder is different from Simulator or device. |
```
Comment (tbaranes) [inline Documentation/APIs.md]: Didn't that link shortcut, is it working only for the organisations?

Comment (JakeLin) [inline Documentation/APIs.md]: The url is not right, I will fix it now.

**Comment by JakeLin** [comment]
@tbaranes I have fixed those URLs, we can use relevant URLs in markdown files.

**Comment by tbaranes** [comment]
It seems good 🤔
Right now, I don't see anything missing, but I will keep you in touch if I have any ideas.

**Comment by tbaranes** [comment]
@JakeLin Merge this whenever you think it's ready, can't think to anything else 👍

**Comment by JakeLin** [comment]
@tbaranes Sure, will do it tomorrow night, I need some time to fix it if something happens.

**Pull request merged**